src/libcore/bool/mod.rs

```diff
@@ -0,0 +1,47 @@
+//! impl bool {}
+
+#![stable(feature = "core_bool", since = "1.39.0")]
```

Comment:
Not sure if this is necessary after changing to `mod bool;` but if possible, remove this / change to `unstable`.